server/db/driver/pg/accounts.go

```diff
@@ -30,6 +30,19 @@ func (a *Archiver) CloseAccount(aid account.AccountID, rule account.Rule) error
 	return nil
 }
 
+// OpenAccount opens the account by setting the value of the rule column.
+func (a *Archiver) OpenAccount(aid account.AccountID) error {
+	err := setRule(a.db, a.tables.accounts, aid, account.NoRule)
+	if err != nil {
+		// fatal unless 0 matching rows found.
+		if !errors.Is(err, errNoRows) {
+			a.fatalBackendErr(err)
+		}
+		return fmt.Errorf("error opening account %s: %v", aid, err)
+	}
+	return nil
```

Comment:
Please factor out this into `(a *Archiver).setAccountRule` and call it from both `CloseAccount` and `OpenAccount` or whatever we call it.  You could even just call `CloseAccount(aid, account.NoRule)` from `OpenAccount`, but that might be confusing to read.

Reply:
You do still want the unexported `setRule` function, correct?

Reply:
Yeah.